public/app/features/alerting/unified/state/actions.ts

```diff
@@ -133,72 +128,10 @@ export const fetchPromRulesAction = createAsyncThunk(
 
 export const fetchAlertManagerConfigAction = createAsyncThunk(
```

Comment:
Are we going to get rid of this action eventually?

Reply:
Yep, forgot about removing it in this PR – will amend